.github/workflows/linux.yml

```diff
@@ -186,7 +187,7 @@ jobs:
           username: ${{ github.repository_owner }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
-      - if: github.ref == 'refs/heads/main'
+      - if: startsWith(github.ref, 'refs/tags/')
```

Comment:
Will `github.ref` have the correct value here? Reason I ask is that release_artifacts.yml is invoked through `workflow_call` as a workaround and not `publish`, since the `hub release create` command is running with a `GITHUB_TOKEN`